Submodule plugins/obs-browser contains modified content
diff --git a/plugins/obs-browser/obs-browser-plugin.cpp b/plugins/obs-browser/obs-browser-plugin.cpp
index d035baa..66c9389 100644
--- a/plugins/obs-browser/obs-browser-plugin.cpp
+++ b/plugins/obs-browser/obs-browser-plugin.cpp
@@ -324,7 +324,7 @@ static void BrowserInit(void)
 	prod_ver << std::to_string(obs_maj) << "." << std::to_string(obs_min)
 		 << "." << std::to_string(obs_pat);
 
-#if CHROME_VERSION_BUILD >= 4472
+#if CHROME_VERSION_BUILD >= 0
 	CefString(&settings.user_agent_product) = prod_ver.str();
 #else
 	CefString(&settings.product_version) = prod_ver.str();
